contracts/identity/AlastriaIdentityManager.sol

```diff
@@ -85,13 +85,13 @@ contract AlastriaIdentityManager is AlastriaIdentityServiceProvider, AlastriaIde
 
     /**
     * @dev Creates a new AlastriaProxy contract for an owner and recovery and allows an initial forward call which would be to set the registry in our case
-    * @param publicKey to be encoded in PublicKeyRegistry call from IdentityManager
+    * @param publicKeyHash to be encoded in PublicKeyRegistry call from IdentityManager
     */
-    function createAlastriaIdentity(string memory publicKey, uint256 format) public validAddress(msg.sender) isPendingAndOnTime(msg.sender) {
+    function createAlastriaIdentity(bytes32 publicKeyHash) public validAddress(msg.sender) isPendingAndOnTime(msg.sender) {
```

Comment:
@alejandroalffer Do you remember why uint256 format was passed? If I remember correctly, the idea was to indicate the format of the public key, for example, FORMAT {JWK, PEM, DER, RAW} although then the function addPublicKey only receives a bytes32...

Reply:
If we all agree to remove it since it is not being used in the addPublicKey function, the change would be correct.

Reply:
Thats the idea, the format of the key is important for the crypto libraries, if we change to a key hash, it is not necessary, in the JOSE header of the JWT the key MUST be specified.